Firebase/Database/third_party/FImmutableSortedDictionary/LICENSE

```diff
@@ -0,0 +1,21 @@
+Copyright (c) 2012 Mads Hartmann Jensen
+ +
```

Comment:
Can you remove the extra "+" here and below? Thanks!

Reply:
Done. Thanks!